docs/data-sources/maintenance_window.md

```diff
@@ -8,6 +8,8 @@ subcategory: "Maintenance Windows"
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
 
+-> **NOTE:** Maintenance window times use the project's configured timezone. To change the timezone, update the Project Time Zone setting in the Atlas Project Settings.
```

Comment:
question: can users configure this with the terraform provider? if so it would be great to link to that page.